ajax/libs/jquery.progressStep/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "jquery.progressStep",
+  "filename": "progressStep.min.js",
+  "version": "1.0.3",
+  "description": "Dynamically renders a progress bar with numbered steps.",
+  "author": {
+    "name": "Matthew Eagar",
+    "url": "https://github.com/mateagar/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/mateagar/progressStep.git"
+  },
+  "keywords": [
+    "progress",
+    "bar",
+    "step",
+    "navigation"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/mateagar/progressStep.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "progressStep*"
```

Comment:
This pattern will get the `progressStep.jquery.json` file which we can't host.
Please try the correct pattern to get the files which are what we want.
Thanks.